torch/custom_libraries.py

```diff
@@ -0,0 +1,52 @@
+import torch
+
+existing_libraries = {'aten', 'quantized'}
```

Comment:
This doesn't seem like the right strategy for matching against existing libraries. In particular, if you are loading a C++ extension, you could end up with extra libraries that are preexisting. Instead, it seems like we should rely on the C++ itself on preventing duplicate libraries (and I think we already have the logic to do this)